net/portmapper/portmapper_test.go

```diff
@@ -74,7 +74,7 @@ func TestProbeIntegration(t *testing.T) {
 	})
 
 	c.SetGatewayLookupFunc(func() (gw, self netaddr.IP, ok bool) {
-		return netaddr.IPv4(127, 0, 0, 1), netaddr.IPv4(1, 2, 3, 4), true
+		return netaddr.IPv4(0, 0, 0, 0), netaddr.IPv4(1, 2, 3, 4), true
```

Comment:
uh, we can dial 0.0.0.0 ?

Reply:
yea not totally sure how this works... I was just permuting as many things as possible